README.md

```diff
@@ -248,16 +235,15 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 
 **Python**
 - [ARTIQ](https://github.com/m-labs/artiq) - Next-generation control system for quantum information experiments.
+- [OLSQ](https://github.com/tbcdebug/OLSQ) - OpenQASM package to perform optimal layout synthesis for quantum computing.
```

Comment:
somehow this was added twice - to Experimental Quantum Computing and Quantum Control.

Reply:
After looking at the project in more detail I would put it into the "Quantum Compiler" section since it is about compiling/rewriting a circuit for a specific QPU architecture